shared/my-sites/themes/theme-options.js

```diff
@@ -64,9 +64,15 @@ export default function getButtonOptions( site, theme, actions, setSelectedTheme
 	}
 };
 
-function rawOptions( site, theme ) {
+function rawOptions( site, theme, isLoggedOut ) {
 	return [
 		{
+			name: 'signup',
+			label: i18n.translate( 'Start a Blog' ),
```

Comment:
Yeah, that's hard. I struggled myself for a while. Let's start with "Choose this design" then, and we can iterate.